libcsm/bss/api.py

```diff
@@ -29,6 +29,9 @@
 import requests
 
 from libcsm import api
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
We should not disable this.  Something like this will allow a user to bypass it if they want, but will use the default cert path and use that if it is available.
```
crt_path = getenv("REQUESTS_CA_BUNDLE", certifi.where())
req = reqests.get("https://something", verify=crt_path)
```

Reply:
fixed